client/my-sites/pages/page-list.jsx

```diff
@@ -75,14 +76,27 @@ export default class PageList extends Component {
 			page: this.state.page,
 			number: 20, // all-sites mode, i.e the /me/posts endpoint, only supports up to 20 results at a time
 			search,
-			status: mapStatus( status ),
+			// When searching, search across all statuses so the user can
+			// always find what they are looking for, regardless of what tab
+			// the search was initiated from. Use POST_STATUSES rather than
+			// "any" to do this, since the latter excludes trashed posts.
+			status: search ? POST_STATUSES.join( ',' ) : mapStatus( status ),
```

Comment:
It might make sense to rename `POST_STATUSES` to something more appropriate as it is shared between pages and posts now?

Reply:
This refers to general post statuses in WordPress. For example pages, and other custom post types like feedbacks or testimonials are all still "posts". So the naming should be okay here for now 👍 

https://wordpress.org/support/article/post-types/